cdk/requirements.txt

```diff
@@ -61,14 +61,14 @@ aws-cdk.cx-api==1.129.0
 aws-cdk.lambda-layer-awscli==1.129.0
 aws-cdk.pipelines==1.129.0
 aws-cdk.region-info==1.129.0
-awscli==1.18.179
-botocore==1.19.19
+botocore==1.24.0
 cattrs==1.8.0
 cffi==1.15.0
 colorama==0.4.3
 constructs==3.3.161
 cryptography==35.0.0
 docutils==0.15.2
+importlib-resources==5.4.0
```

Comment:
I see in [pypi](https://pypi.org/project/importlib-resources/#description) that:
> importlib_resources is a backport of Python standard library [importlib.resources](https://docs.python.org/3/library/importlib.html#module-importlib.resources) module for older Pythons.

What are you using this for?

Reply:
Good question. Let me see if I can track down what caused it to be installed.